### PR TITLE
Migrated Batch 1 of Density data

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: src/data/density-constants.ts [hunk 1]
```diff
@@ -15,6 +15,490 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
     densityAtMinimumTemperature: 21.423,
     densityAtMaximumTemperature: 6.4935,
   },
+
+  //<----Batch1
```
Comment (sumitnegi7) [inline src/data/density-constants.ts:19]: Comments can be removed ....rest looks good :+1:

**Review by sumitnegi7 — APPROVED** [review]

**Comment by sumitnegi7** [comment]
@saikat-samanta  please verify the data once.

**Review by saikat-samanta — APPROVED** [review]
LGTM 👍

File: src/data/density-constants.ts (hunk 1 above)
Comment (maneetgoyal) [inline src/data/density-constants.ts:19]:
```suggestion
```

File: src/data/density-constants.ts [hunk 2]
```diff
@@ -15,6 +15,490 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
...
+    C2: 0.24967,
+    C3: 676,
+    C4: 0.22023,
+    minimumTemperature: 196.15,
+    maximumTemperature: 676,
+    densityAtMinimumTemperature: 11.872,
+    densityAtMaximumTemperature: 3.2786,
+  },
+
+  //Batch1 --->
```
Comment (maneetgoyal) [inline src/data/density-constants.ts:500]:
```suggestion
```

Comment (maneetgoyal) [inline src/data/density-constants.ts:500]:
```suggestion
```

**Pull request merged**